server/storage/rethinkdb_models.go

```diff
@@ -6,9 +6,10 @@ import (
 
 // These consts are the index names we've defined for RethinkDB
 const (
-	rdbSha256Idx        = "sha256"
-	rdbGunRoleIdx       = "gun_role"
-	rdbGunRoleSha256Idx = "gun_role_sha256"
+	rdbSha256Idx         = "sha256"
+	rdbGunRoleIdx        = "gun_role"
+	rdbGunRoleSha256Idx  = "gun_role_sha256"
+	rdbGunRoleVersionIdx = "gun_role_version"
```

Comment:
Rethink can't reuse the same name for secondary indexes as it does for primary key (failure message here: https://circleci.com/gh/docker/notary/3840#tests/containers/3) -- can we just use the primary key?